python_ta/cfg/graph.py

```diff
@@ -274,11 +277,32 @@ def update_edge_z3_constraints(self) -> None:
 
                 # traverse into target node
                 for node in edge.target.statements:
-                    if isinstance(node, Assign):
```

Comment:
This change should be extracted into a separate pull request. It is separate from checking edge feasibility; it is extending the existing capability of the Z3 constraint updating, which is a good change to make but is separate.

In your new PR, you should make sure to add an entry to the changelog describing this, as well as add new test cases for just this change.